S/Spot_julia/build_tarballs.jl

```diff
@@ -89,9 +85,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("libcxxwrap_julia_jll"),
-    BuildDependency("libjulia_jll")
+    BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.9")),
+    Dependency("libcxxwrap_julia_jll")
```

Comment:
You should add a compat for `libcxxwrap_julia_jll`, depending on whatever version of CxxWrap.jl you want to to match.